src/Comonad.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+var assert = require('./internal/assert');
+var compose = require('./internal/compose');
+var extend = require('./internal/extend');
+var extract = require('./internal/extract');
+
+
+module.exports = function(equals) {
+  return {
+
+    //  extend extract w = w
+    TK1: assert.forall1(function(w) {
```

Comment:
this laws are same as ones from: [Control/Comonad.purs](https://github.com/purescript/purescript-control/blob/f14399dcd657739745de74cc2cf4ebc3595aadee/src/Control/Comonad.purs#L18-L19)
So i guess it's Left Identity and the other is Right Identity. 
(we need to add names to FL too)